qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -63,6 +63,8 @@ class CephFSTestCase(CephTestCase):
     def setUp(self):
         super(CephFSTestCase, self).setUp()
 
+        self.config_set('mon', 'mon_allow_pool_delete', True)
```

Comment:
This looks like the real fix. Why do we need the other changes? (a separate commit + explanation would be helpful)

Reply:
Yes, that's correct. This is sufficient to fix the traceback.
The other changes doesn't have functional impact. I have split the commit into two and tried explaining there. Hope that helps.